### DISCUSSION TITLE
Failed ctests -M in clean install of Ubuntu 22.04 and in WSL running 22.04LTS

### DISCUSSION THREAD

**Discussion by mechlin** [neperfepx/neper#917] [Q&A] [answered]
Having issues with the latest Neper version 4.10.0 failing make tests (ctest -R M) for meshing components. I've tried both latest stable gmsh version 4.14.0 and 4.11.1 compiled from source. I've also tried apt-get gmsh. Tried this on both WSL and a clean Ubuntu 22.04 LTS install. Running g++ 11.4.0. Also tried with and without libfltk1.3dev / libfltk1.3 installed via apt-get for gmsh compile from source. Anybody else having ctest failures? Running "neper -T -n 1" works fine. 

 147 - M/cl (Failed)
        148 - M/cl_expr (Failed)
        151 - M/clmin (Failed)
        152 - M/clratio (Failed)
        153 - M/cut (Failed)
        155 - M/dim2 (Failed)
        156 - M/dim_expr (Failed)
        158 - M/faset (Failed)
        161 - M/interface (Failed)
        162 - M/interface2 (Failed)
        164 - M/mesh2dalgo_dela (Failed)
        165 - M/mesh2dalgo_fron (Failed)
        166 - M/mesh2dalgo_mead (Failed)
        167 - M/mesh2dalgo_netg (Failed)
        168 - M/mesh3dalgo_netggmne (Failed)
        169 - M/mesh3dalgo_netggmsh (Failed)
        170 - M/mesh3dalgo_netgnetg (Failed)
        171 - M/meshing (Failed)
        173 - M/meshqualdisexpr (Failed)
        174 - M/meshqualexpr (Failed)
        175 - M/meshqualmin (Failed)
        176 - M/morpho_cube (Failed)
        177 - M/msh4 (Failed)
        178 - M/nset1 (Failed)
        179 - M/nset2 (Failed)
        180 - M/nset3 (Failed)
        181 - M/order (Failed)
        188 - M/pl (Failed)
        189 - M/rcl (Failed)
        190 - M/rcl_expr (Failed)
        193 - M/remesh1 (Failed)
        194 - M/remesh2 (Failed)
        195 - M/remesh3 (Failed)
        197 - M/rodrigues_per_cubic (Failed)
        198 - M/rodrigues_per_hexagonal (Failed)
        204 - M/statelsetco (Failed)
        210 - M/stateltco (Failed)

**Comment by rquey** [comment, marked as answer]
Fixed it [today](https://github.com/neperfepx/neper/commit/44911dde69472fd6e7afcd87c1ddc0a490843394). Please give a try to the latest version (4.10.1-1).

More specifically, this bug fix was introduced for compatibility with Gmsh 4.13.1, i.e. the latest official version. I can also confirm compatibility with versions 4.12.2 and 4.11.1...

**Reply by mechlin** [reply]
Thanks Romain. Works perfectly now!

**Reply by rquey** [reply]
Great. I've just made a new version.
